aries-site/src/pages/templates/inline-notifications.mdx

```diff
@@ -72,7 +72,7 @@ for the following scenarios:
 ## Anatomy
 
 <Example plain>
-  <InlineNotificationAnatomy width="large" />
+  <InlineNotificationAnatomy width="xlarge" />
```

Comment:
It looks like InlineNotificationAnatomy doesn't accept a width, wondering if we should just change this to
```suggestion
  <InlineNotificationAnatomy />
```